src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -23,6 +24,59 @@ public class RaspberryPi3Driver : GpioDriver
         private readonly Set_Register _setClearRegister;
         private readonly Get_Register _getClearRegister;
 
+        /// <summary>
+        /// Used to set the Alternate Pin Mode on Raspberry Pi 3/4.
+        /// The actual pin function for anything other than Input or Output is dependent
+        /// on the pin and can be looked up in the Raspi manual.
+        /// </summary>
+        public enum AltMode
+        {
+            /// <summary>
+            /// The mode is unknown
+            /// </summary>
+            Unknown,

```

Comment:
Since you never return this or allow this to set, perhaps consider removing this value

Reply:
I found that it will be helpful for the client, therefore I added it.